test/controllers/issueish-list-controller.test.js

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import {shallow} from 'enzyme';
+
+import {createPullRequestResult} from '../fixtures/factories/pull-request-result';
+import Issueish from '../../lib/models/issueish';
+import {BareIssueishListController} from '../../lib/controllers/issueish-list-controller';
+
+describe('IssueishListController', function() {
```

Comment:
This component has an error state too, yes?  Would that be worth testing?

Reply:
Done 👍